tests/general/InvalidLinkBearTest.py

```diff
@@ -229,3 +230,35 @@ def test_links_to_ignore(self):
 
         self.assertResult(valid_file=valid_file,
                           settings={'link_ignore_list': link_ignore_list})
+
+    def test_variable_timeouts(self):
+        nt = {
+            'https://google.com/timeout': 10,
+            'https://facebook.com/timeout': 15
+        }
+
+        file_contents = """
+        https://facebook.com/
+        https://google.com/
+        https://coala.io/
+        """.splitlines()
+
+        def response(status_code, *args, **kwargs):
+            res = requests.Response()
+            res.status_code = status_code
+            return res
+
+        with unittest.mock.patch(
+                   'tests.general.InvalidLinkBearTest.requests.head',
+                   return_value=response(status_code=200)) as mock:
```

Comment:
What is the preferred indentation in this case?
*two tabs or three tabs* relative to the `with` block?